tests/test_variety.py

```
@@ -0,0 +1,184 @@
+"""Tests for the weekly outfit variety score service."""
+
+from datetime import datetime, timedelta, timezone
+from unittest.mock import MagicMock
+
+from server.services.variety import compute_variety_score
+
+
+def _wear_row(items, hours_ago):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_wear_row`

(ANN202)